test/e2e/util/cluster/aws/cluster.go

```diff
@@ -86,7 +86,7 @@ func DumpHostedCluster(t *testing.T, ctx context.Context, hc *hyperv1.HostedClus
 			LogCheckers: []aws.LogChecker{findKubeObjectUpdateLoops},
 		})
 		if err != nil {
-			t.Errorf("Failed to dump cluster: %v", err)
```

Comment:
I don't think we should ignore errors here, this functionality is important and we run some verifications over the dumped data. If we ignore failures here, we allow introducing regressions.
/hold

Reply:
We can take up this piece separately; removed from the PR